src/aics-image-viewer/shared/constants.ts

```diff
@@ -98,19 +98,37 @@ export const PRESET_COLOR_MAP = Object.freeze([
 /** Allows the 3D viewer to apply the default camera settings for the view mode. */
 const USE_VIEW_MODE_DEFAULT_CAMERA = undefined;
 
+const viewModeToDefaultCameraPosition: Record<ViewMode, [number, number, number]> = {
+  [ViewMode.threeD]: [0, 0, 5],
+  [ViewMode.xy]: [0, 0, 2],
+  [ViewMode.xz]: [0, 2, 0],
+  [ViewMode.yz]: [2, 0, 0],
+};
+
+const viewModeToDefaultCameraUp: Record<ViewMode, [number, number, number]> = {
+  [ViewMode.threeD]: [0, 1, 0],
+  [ViewMode.xy]: [0, 1, 0],
+  [ViewMode.xz]: [0, 0, 1],
+  [ViewMode.yz]: [0, 0, 1],
+};
```

Comment:
these really look like they should be coming from volume-viewer but I suppose they are fine here.